app/src/main/java/ru/iteco/fmhandroid/api/ApiModule.kt

```diff
@@ -31,4 +32,11 @@ object ApiModule {
         return retrofit
             .create(NewsApi::class.java)
     }
+
+    @Provides
+    @Singleton
+    fun provideNurseStationApi(@Authorized retrofit: Retrofit): NurseStationApi {
+        return retrofit
+            .create(NurseStationApi::class.java)
+    }

```

Comment:
Там вроде вот так можно было сделать через расширение котлина
```suggestion
    fun provideNurseStationApi(@Authorized retrofit: Retrofit) = retrofit.create<NurseStationApi>()
```

Reply:
Так вроде все в одном стиле